plugins/woocommerce/src/Admin/Overrides/OrderTraits.php

```diff
@@ -80,4 +88,24 @@ public function get_item_shipping_tax_amount( $item ) {
 	public function get_item_coupon_amount( $item ) {
 		return floatval( $item->get_subtotal( 'edit' ) - $item->get_total( 'edit' ) );
 	}
+
+	/**
+	 * Calculate cart tax amount for line item/product.
+	 *
+	 * @param WC_Order_Item $item Line item from order.
+	 *
+	 * @return float|int
```

Comment:
I guess this happens in other functions in this class, but if we initialize as `0.0` then we can always return a float.